src/validinput.js

```diff
@@ -0,0 +1,31 @@
+module.exports = [
+        'a',
+        'b',
+        'c',
+        'd',
+        'e',
+        'f',
+        'g',
+        'h',
+        'i',
+        'j',
+        'k',
+        'l',
+        'm',
+        'n',
+        'o',
+        'p',
+        'q',
+        'r',
+        's',
+        't',
+        'u',
+        'v',
+        'w',
+        'x',
+        'y',
+        'z'
+    ]
+    
+
+]
```

Comment:
This syntax error prevented your entire game from running. Had you ran your code and used the error message to see where the error is occurring you would have passed the entirety of the requirements.